tests/lite/test_lite.py

```diff
@@ -164,57 +164,6 @@ def test_setup_dataloaders_return_type():
     assert lite_dataloader1.dataset is dataset1
 
 
-def test_setup_custom_dataloaders():
```

Comment:
We could have a custom dataloader using only a dataset as a test.

Reply:
We already test that in `tests/trainer/test_data_loading.py`

Reply:
I meant, we should unittest the context manager to make sure the params are

Reply:
@tchaton isn't that covered by the test that @awaelchli added in https://github.com/PyTorchLightning/pytorch-lightning/pull/10334?